specs/fulu/validator.md

```diff
@@ -88,24 +90,29 @@ each other, with extra fields for the necessary context.
 
 The sequence of sidecars associated with a block and can be obtained by first computing
 `cells_and_kzg_proofs = [compute_cells_and_kzg_proofs(blob) for blob in blobs]` and then calling
-`get_data_column_sidecars(signed_block, cells_and_kzg_proofs)`.
+`get_data_column_sidecars_from_block(signed_block, cells_and_kzg_proofs)`.
+
+Moreover, the full sequence of sidecars can also be computed from `cells_and_kzg_proofs` and any single
+`data_column_sidecar`, by calling `get_data_column_sidecars_from_column_sidecar(sidecar, cells_and_kzg_proofs)`.
```

Comment:
One more nit, the variable "data_column_sidecar" at the beginning of the sentence doesn't match the first argument "sidecar" to the function. I think these should be the same.

Reply:
Updated the `data_column_sidecar` here to just `sidecar to match the function, and removed the blank line below. Thanks!